paper.md

```diff
@@ -1,32 +1,150 @@
 ---
-title: 'sourmash: a library for MinHash sketching of DNA'
+title: 'sourmash: a tool to quickly search, compare, and analyze genomic and metagenomic data sets'
 tags:
+  - FracMinHash
   - MinHash
   - k-mers
   - Python
+  - Rust
 authors:
- - name: C. Titus Brown
-   orcid: 0000-0001-6001-2677
-   affiliation: University of California, Davis
  - name: Luiz Irber
    orcid: 0000-0003-4371-9659
    affiliation: University of California, Davis
-date: 13 Sep 2016
+ - name: N. Tessa Pierce-Ward
+   orcid: 0000-0002-2942-5331
+   affiliation: University of California, Davis
+ - name: Mohamed Abuelanin
+   orcid: 0000-0002-3419-4785
+   affiliation: University of California, Davis
+ - name: Harriet Alexander
+   orcid: 0000-0003-1308-8008
+   affiliation: Woods Hole Oceanographic Institute
+ - name: Abhishek Anant
+   orcid:  0000-0002-5751-2010
+   #affiliation: @@@
+ - name: Keya Barve
+   orcid: 0000-0003-3241-2117
+   affiliation: University of California, Davis
+ - name: Colton Baumler
+   orcid: 0000-0002-5926-7792
+   affiliation: University of California, Davis
+ - name: Olga Botvinnik
+   orcid: 0000-0003-4412-7970
+   affiliation: Bridge Bio
+ - name: Phillip Brooks
+   orcid: 0000-0003-3987-244X
+   affiliation: Ancestry
+ - name: Peter Cock
+   orcid: 0000-0001-9513-9993
+   #affiliation: @@@
+ - name: Daniel Dsouza
+   orcid: 0000-0001-7843-8596
+   #affiliation: @@@
+ - name: Laurent Gautier
+   orcid: 0000-0003-0638-3391
+   #affiliation: @@@
+ - name: Nick H
+   orcid: 0000-0002-1685-302X
+   #affiliation: @@@
+ - name: Tim Head
+   orcid: 0000-0003-0931-3698
+   #affiliation: @@@
+ - name: Mahmudur Rahman Hera
+   orcid: 0000-0002-5992-9012
+   affiliation: Pennsylvania State University
+ - name: Hannah Eve Houts
+   orcid: 0000-0002-7954-4793
+   affiliation: University of California, Davis
+ - name: Lisa K. Johnson
+   orcid: 0000-0002-3600-7218
+   affiliation: 10x Genomics
+ - name: Fabian Klötzl
+   orcid: 0000-0002-6930-0592
+   #affiliation: @@@
+ - name: David Koslicki
+   orcid: 0000-0002-0640-954X
+   affiliation: Pennsylvania State University
+ - name: Katrin Leinweber
+   orcid: 0000-0001-5135-5758
+   #affiliation: @@@
+ - name: Marisa Lim
+   orcid: 0000-0003-2097-8818
+   affiliation: 10x Genomics
+ - name: Ricky Lim
+   orcid: 0000-0003-1313-7076
+   #affiliation: @@@
```

Comment:
hi @ricky-lim, I am updating the list of sourmash authors to release a JOSS paper for version 4.x. Can you please reply with your desired affiliation? Thanks!

(Of course, also please let me know if you'd prefer not to be included as an author.)